Screenbox/Controls/ChapterProgressBar.xaml

```diff
@@ -18,11 +18,11 @@
 
             <DataTemplate x:Key="ItemTemplate" x:DataType="viewModels:ChapterViewModel">
```

Comment:
Instead of spamming `Mode=OneWay` I could have just done:

```xml
<DataTemplate
    x:Key="ItemTemplate"
    x:DataType="viewModels:ChapterViewModel"
    x:DefaultBindMode="OneWay">
```